File: app/src/main/java/in/mvpstarter/sample/util/rx/scheduler/TrampolineMainScheduler.java
package in.mvpstarter.sample.util.rx.scheduler;

import io.reactivex.android.schedulers.AndroidSchedulers;
import io.reactivex.schedulers.Schedulers;

/**
 * Created by lam on 2/6/17.
 */

public class TrampolineMainScheduler<T> extends BaseScheduler<T> {

    protected TrampolineMainScheduler() {
        super(Schedulers.trampoline(), AndroidSchedulers.mainThread());
    }
}

File: app/src/main/java/io/mvpstarter/sample/util/rx/scheduler/TrampolineMainScheduler.kt
package io.mvpstarter.sample.util.rx.scheduler

import io.reactivex.android.schedulers.AndroidSchedulers
import io.reactivex.schedulers.Schedulers

/**
 * Created by lam on 2/6/17.
 */

class TrampolineMainScheduler<T> private constructor() : BaseScheduler<T>(Schedulers.trampoline(), AndroidSchedulers.mainThread())
